### ISSUE TITLE
Continuing Maintenance?

### ISSUE THREAD

**Issue by JPeisach** [raspberrypi/maynard#56]
Is Maintenance still going on here? Is this project dead?

**Comment by ZanderBrown** [comment]
It's very dead

**Comment by JPeisach** [comment]
Is there any plans to revive it or something?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by andrewshadura** [comment]
You may be interested in https://gitlab.apertis.org/hmi/maynard.

**Comment by hyyoxhk** [comment]
@andrewshadura No, it's not for weston compositor, the interface is "agl_shell" and "agl_shell_desktop", not is "weston_desktop_shell"
